imports/server/mediasoup.ts

```diff
@@ -705,7 +713,9 @@ class SFU {
       return;
     }
 
-    transport.observer.on(
+    const wtransport = transport as types.WebRtcTransport;
```

Comment:
Ugh. I'm annoyed that they made their code all Java so there's no way to do this with type narrowing

Reply:
Yeah I looked to see if they had type maps or anything hiding around but didn't find anything